letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -281,6 +281,26 @@ BootstrapRpmCommon() {
     exit 1
   fi
 
+  if [ "$ASSUME_YES" = 1 ]; then
+    yes_flag="-y"
+  fi
+
+  if ! $SUDO $tool list *virtualenv > /dev/null 2>&1; then
+    echo "To use Certbot, packages from the EPEL repository need to be installed."
+    if [ "$ASSUME_YES" = 1 ]; then
+        /bin/echo -n "Enabling the EPEL repository in 3 seconds..."
+        sleep 1s
```

Comment:
`sleep` on Fedora etc is GNU sleep, which supports the `s` (as well as `m` and `h`).